### DISCUSSION TITLE
Switch to Alpine sort plugin for v3

### DISCUSSION THREAD

**Discussion by binaryfire** [mokhosh/filament-kanban#61] [Ideas]
I just noticed Filament is moving to the Alpine sort plugin for v4: https://github.com/filamentphp/filament/issues/12666

The sort plugin is just a wrapper for SortableJS so it has all the same functionality: https://alpinejs.dev/plugins/sort

I think keeping the code style as close as possible to Filament core will make it easier for other devs to contribute to your plugin in the future.

**Comment by mokhosh** [comment]
Sure, I had this in mind since Caleb started playing with the sort plugin 🙏
